CHANGELOG

```diff
@@ -3,6 +3,51 @@ devel
 
 * Added AQL bit handling functions BIT_AND, BIT_OR, BIT_XOR and BIT_COUNT.
 
+* Fixed some situations of
+  [...]
+  SUBQUERY
+  FILTER
+  LIMIT
+  [...]
+  in AQL queries, yielding incorrect responses. A distributed
+  state within the subquery was not resetted correctly.
```

Comment:
```suggestion
  state within the subquery was not reset correctly.
```

Reply:
Unrelated to the changes in this PR, but I fixed it in here as well.